catalog/leap-ingested/large_ensemble_pco2_testbed.yaml

```diff
@@ -0,0 +1,31 @@
+title: Large ensemble pCO2 testbed
+description: Large ensemble pCO2 testbed (manually ingested)
+maintainers:
+  - name: Julius Busecke
+    github: jbusecke
+tags:
+  - climate
+  - oceanography
+  - pco2
+  - carbon
+  - zarr
+provenance:
+  providers:
+    - name: CESM
+      description: Community Earth System Model
+      url: "https://www.cesm.ucar.edu/models"
+    - name: CanESM2
+      description: Canadian Earth System Model 2
+      url: "https://www.canada.ca/en/environment-climate-change/services/climate-change/science-research-data/modeling-projections-analysis/centre-modelling-analysis/models/second-generation-earth-system-model.html"
+    - name: GFDL
+      description: Geophysical Fluid Dynamics Laboratory
+      url: "https://www.gfdl.noaa.gov/"
+  license: CC-BY
+
+stores:
+  - name: CESM.zarr
+    url: "gs://leap-persistent-ro/data-library-manual/CESM.zarr"
+  - name: CanESM2.zarr
+    url: "gs://leap-persistent-ro/data-library-manual/CanESM2.zarr"
+  - name: GFDL.zarr
+    url: "gs://leap-persistent-ro/data-library-manual/GFDL.zarr"
```

Comment:
this is an example of a feedstock containing datasets produced outside of the pangeo-forge workflow. for this type of feedstock, we need to define the required metadata

Reply:
I am a bit confused about these yaml files here. Are these supposed to be entered by the user? Or are they built by the CI here? 

I was under the assumption that the user would just provide a link to a feedstock directory and our logic then pulls info from both `meta.yaml` and `catalog.yaml` to construct [the JSON file](https://github.com/leap-stc/data-management/blob/revamped-catalog/catalog/consolidated-web-catalog.json)? 

I personally like this, since the user would only have to add a single line to add their feedstock to the catalog and then could make all the required changes in their repository (by editing the catalog.yaml and meta.yaml)?